src/Sylius/Bundle/ApiBundle/OpenApi/Documentation/StatisticsDocumentationModifier.php

```diff
@@ -37,42 +39,46 @@ public function modify(OpenApi $docs): OpenApi
         $schemas['Statistics'] = [
             'type' => 'object',
             'properties' => [
-                'salesPerPeriod' => [
+                'sales' => [
                     'type' => 'array',
                     'items' => [
                         'type' => 'object',
                         'properties' => [
                             'period' => [
                                 'type' => 'string',
                                 'format' => 'date-time',
+                                'example' => '1999-12',
                             ],
                             'total' => [
                                 'type' => 'integer',
                                 'example' => '1000',
```

Comment:
```suggestion
                                'example' => 1000,
```
as it is an integer?